.lingua

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Don't you need history module rewritten first since its a child module of add layers @jmacura ?

Reply:
@jmacura Must have gotten by mistake, it shouldn't exist at all, it is included in the .gitignore file

Reply:
@raitisbe You are correct, yet there is @output decoration included inside history-list and add layers, have to use it properly for everything to work together. That is my theory.

Reply:
> 
> 
> @jmacura Must have gotten by mistake, it shouldn't exist at all, it is included in the .gitignore file

When you accidentally commit the file before you mark it in .gitignore, then it will stay in the repo and won't get deleted until you do so manually. It has happened to me before.

Reply:
@jmacura Actually it was added to .gitignore before, just in different branch :D I must have created it again in this branch and didn't notice it.

Reply:
@DailisLangovskis Very well! This is exactly how it should look like (you can compare with the 1.24 version here https://ng.hslayers.org/examples/full/).

Reply:
@jmacura Thing is, I didn't change anything in my code

Reply:
@DailisLangovskis It works for me now as well. I think I have tested it before the a08542c , is that possible?
Anyway, I think it's OK to merge now (what about the .lingua file? I think it has to be deleted manually from the GitHub site.).

Side note (@fzadrazil): Is the history list even useful? The browser has its own history for the field, so we are kind of duplicating a functionality that today works out of the box. Maybe we could use it for some list of pre-defined OWS services (per-project)? Just an idea for future enhancement...

Reply:
@jmacura Mby so :) .lingua file is not showing for me on GitHub no more.

Reply:
> Side note (@fzadrazil): Is the history list even useful? The browser has its own history for the field, so we are kind of duplicating a functionality that today works out of the box. Maybe we could use it for some list of pre-defined OWS services (per-project)? Just an idea for future enhancement...

@jmacura I would keep it there so we are not relying on the browser. The autofill feature can be turned off, it does not have to work properly on all devices/browsers, etc. The idea of predefined services is good...but I think that is the purpose of Micka/Layman where those services should be stored.